lua/spectre_common.lua

```diff
@@ -72,15 +72,13 @@ end
 local function determine_if_cacheable(url, namespace, request_headers)
     local cacheability_info = {
         is_cacheable = false,
-        ttl = nil,
-        pattern = nil,
+        cache_entry = {
```

Comment:
Shouldn't this contain all of the other keys? You've got `ttl`, but it seems you also need `bulk_support`, `id_identifier`, `pattern`, etc.

Reply:
Hmmmm, yeah you're right, I can see some readability benefits for defining defaults for all keys.